clients/elrond/dataGetter.go

```diff
@@ -72,10 +79,12 @@ func (dg *elrondClientDataGetter) ExecuteQueryReturningBytes(ctx context.Context
 
 	response, err := dg.proxy.ExecuteVMQuery(ctx, request)
 	if err != nil {
+		dg.log.Error("while query SC", "request", request, "error", err)
 		return nil, err
 	}
 
 	if response.Data.ReturnCode != okCodeAfterExecution {
+		dg.log.Error("while query SC", "request", request, "response", response)
```

Comment:
Please add some toString converters functions, otherwise we won't be able to properly figure out what was queried and what was responded with.

Reply:
done